.eslintignore

```diff
@@ -1,4 +1,5 @@
 /.git/
+/coverage
```

Comment:
nitpicking but I prefer to use a trailing slash for folders to quickly distinguish them from files.